ubuntudesign/www.ubuntu.com
# Merge /management and /support sections into new /support section

## Done

* combined the /management and /supports section now that landscape.canonical.com is better following [Frankie's instructions](https://docs.google.com/document/d/14Fi5pGzj4c0nRrWuXZTQVLeNzm1BysU3AfqmcP3Dlic/edit#heading=h.t3jihh24f7ks)
* rewrote the 3 remaining pages nearly completely
* created and updated all the redirects possible
* updated the navigation
* removed dead css from /section/_management.scss

## QA

1. go through the support section on http://www.ubuntu.com-update-management-section.demo.haus/support and see that it looks good on S/M/L screens
2. compare to the [copy docs]()https://drive.google.com/drive/folders/0B8feV0jqaac3aGhEaWRFaDlNMDA
3. test that the old urls are redirected

- [x] /management/landscape-features
- [x] /management/working-with-landscape
- [x] /management/return-on-investment
- [x] /management/compliance
- [x] /management/ubuntu-advantage (now /support/plans-and-pricing)

## Issue / Card

[trello](https://trello.com/c/MoLkc9SF)
